docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -86,6 +112,17 @@ The table below lists the data that the Bitbucket integration collects from your
     </td>
```

Comment:
I noticed that for Deployments, we're not specifying what the `timestamp` is defined as (lines 109~110) 👀

Reply:
How is that mapping defined?
"?timestamp_merged?: pull request merged date"?

Reply:
exactly!